### ISSUE TITLE
test

### ISSUE THREAD

**Issue by AnushaMarupaka** [AnushaMarupaka/Test-public#3]
<html>
<body>

name | about | title | labels | assignees
-- | -- | -- | -- | --
Major Incident | Use this template for major incidents resulting in an outage | Incident - [Description of incident] |   |  

<h2 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-weight: 600; font-size: 1.5em; line-height: 1.25; border-bottom: 1px solid var(--color-border-secondary); padding-bottom: 0.3em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-roles" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#roles" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Roles</h2><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><strong style="box-sizing: border-box; font-weight: 600;">Incident Commander/SRE On-call:</strong><span> </span>This person is responsible to handle the incident end to end.</p><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><strong style="box-sizing: border-box; font-weight: 600;">Scribe:</strong><span> </span>This person handles all communications and reports to the required channels and to product management to communicate to customers.</p><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><strong style="box-sizing: border-box; font-weight: 600;">SME(Subject Matter Expert):</strong><span> </span>This person will be from the SRE team in case of Infrastructure related issues or application SME in case of application issues and sometimes both. They will focus primarily on investigating the issue during the incident.</p><h2 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-weight: 600; font-size: 1.5em; line-height: 1.25; border-bottom: 1px solid var(--color-border-secondary); padding-bottom: 0.3em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-step-by-step-response" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#step-by-step-response" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Step by step Response</h2><h3 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-size: 1.25em; font-weight: 600; line-height: 1.25; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-initial-response---5-10m" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#initial-response---5-10m" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Initial Response - (5-10m)</h3><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Once<span> </span><strong style="box-sizing: border-box; font-weight: 600;">Incident Commander</strong><span> </span>receives the alert/issue through Pager Duty or through any other source(slack channel etc.) he/she should judge (see<span> </span><a href="https://confluence.ncr.com/display/SWDO/To+Incident%2C+or+not+to+Incident" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">To Incident, or not to Incident</a>) if this comes under the incident category or it's a known issue. Once we quickly decide if this is an incident do the following tasks.</p><ul class="contains-task-list" style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; padding-left: 2em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li class="task-list-item" style="box-sizing: border-box; list-style-type: none;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Post in<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Consolas, &quot;Liberation Mono&quot;, Menlo, monospace; font-size: 13.6px; background-color: var(--color-markdown-code-bg); border-radius: 6px; margin: 0px; padding: 0.2em 0.4em;">#bsp-incidents-all</code><span> </span>slack channel any information relevant to the alert</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Create a private slack channel to discuss the incident. The private slack channel name should be<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Consolas, &quot;Liberation Mono&quot;, Menlo, monospace; font-size: 13.6px; background-color: var(--color-markdown-code-bg); border-radius: 6px; margin: 0px; padding: 0.2em 0.4em;">inc-&lt;dateofincident yyyymmdd&gt;-&lt;appname&gt;</code><span> </span>for example<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Consolas, &quot;Liberation Mono&quot;, Menlo, monospace; font-size: 13.6px; background-color: var(--color-markdown-code-bg); border-radius: 6px; margin: 0px; padding: 0.2em 0.4em;">inc-20210530-ias</code>.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Create a Webex bridge for incidents. Also, post this in<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Consolas, &quot;Liberation Mono&quot;, Menlo, monospace; font-size: 13.6px; background-color: var(--color-markdown-code-bg); border-radius: 6px; margin: 0px; padding: 0.2em 0.4em;">#bsp-incidents-all</code>.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Use Pagerduty to page the other teammates(Scribe/SME) using this<span> </span><a href="https://ncr-saas.pagerduty.com/escalation_policies#P52AVAS" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">escalation policy</a><span> </span>and appropriate teams based on the component details. Please refer to this<span> </span><a href="https://confluence.ncr.com/pages/viewpage.action?pageId=451401718" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">Monitors list with stakeholders</a><span> </span>for contact details for relevant applications/components. Better to over-page than to feel bad waking someone up.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Once all the required people join,<span> </span><strong style="box-sizing: border-box; font-weight: 600;">Incident Commander</strong><span> </span>must brief about the issue and should take inputs from other teams, SMEs and prepare initial analysis, impact, severity, etc.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Scribe should take the notes of this discussion and prepare the draft for initial communications.</li></ul><h3 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-size: 1.25em; font-weight: 600; line-height: 1.25; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-initial-communication---5-10m" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#initial-communication---5-10m" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Initial Communication - (5-10m)</h3><ul style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; padding-left: 2em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li style="box-sizing: border-box;">Once all the details like initial analysis, impact, severity, and other relevant details are collected by the Scribe, they need to send the communications to the following parties.<ul class="contains-task-list" style="box-sizing: border-box; margin-bottom: 0px; margin-top: 0px; padding-left: 2em;"><li class="task-list-item" style="box-sizing: border-box; list-style-type: none;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">Internal Stakeholders</strong><span> </span>- Please use the #bsp-incidents-all slack channel for internal communication and providing a C.A.N. report by using the format specified<span> </span><a href="https://confluence.ncr.com/display/SWDO/Sample+Templates" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">here</a>. (<em style="box-sizing: border-box;">MOSTLY TECHNICAL</em>)</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">Executive Leadership Team</strong><span> </span>- Please use the ELT template mentioned<span> </span><a href="https://confluence.ncr.com/display/SWDO/Sample+Templates" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">here</a><span> </span>to send a mail communication to Jerry or Mike so that then can send it directly to them once the content is approved. There will be sections specific to the ELT summary which is important for visibility across NCR's leadership. (<em style="box-sizing: border-box;">BUSINESS IMPACT - NON-TECHNICAL</em>)</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">External Stakeholders</strong><span> </span>- Now that we have an initial understanding of the issue, please go to<span> </span><a href="http://statuspage.io/" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">statuspage.io</a><span> </span>to update the incident with details regarding the incident. This needs approval from leadership(Jerry/Jemmy/Mike) prior to submitting. We can make use of ELT Summary we used earlier. For now(until automation is done) updating status page can be done by the Scribe after approval.<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: 600;">Note:</strong><span> </span>We will be providing a fully-fledged RCA after the incident, so please refrain from going too deep here and simply focus on providing business impact. (<em style="box-sizing: border-box;">BUSINESS IMPACT - NON TECHNICAL &amp; NON CONFIDENTIAL</em>)</li></ul></li></ul><h3 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-size: 1.25em; font-weight: 600; line-height: 1.25; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-investigation-and-triage" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#investigation-and-triage" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Investigation and Triage</h3><ul class="contains-task-list" style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; padding-left: 2em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li class="task-list-item" style="box-sizing: border-box; list-style-type: none;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>While initial communication is being sent by the Scribe, the Incident Commander can start troubleshooting the issue with the SME. If there are any updates from the investigation the Incident Commander and SME will update the Scribe with any new information.<br style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: 600;">Note</strong><span> </span>Please see the below link for the severity-based matrix and communication windows.<span> </span><a href="https://confluence.ncr.com/display/ncrsrem/Incident+Severity+Matrix" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">Incident Severity Matrix</a></li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>During this phase, if any metrics need to be collected he/she needs to do it and update it on regular basis and work towards stabilizing the environment rather than finding the root cause by taking actions(with approval from SRE/Leadership). If any tickets need to be created Scribe/CommunicationsLead can help.</li></ul><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">On a regular basis the Incident Commander will<span> </span><strong style="box-sizing: border-box; font-weight: 600;">evaluate the situation</strong>, take help from SMEs, and page out to any other needed teams. They will also ensure the group remains focused on the current incident at hand.</p><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><strong style="box-sizing: border-box; font-weight: 600;">Communications should be sent out every 30 minutes. Add a comment to this issue to reflect when and what communications were sent.</strong></p><h3 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-size: 1.25em; font-weight: 600; line-height: 1.25; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a id="user-content-resolution" class="anchor" aria-hidden="true" href="https://github.com/ncr-bsp/incidents/blob/master/.github/ISSUE_TEMPLATE/major-incident.md#resolution" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; float: left; line-height: 1; margin-left: -20px; padding-right: 4px;"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Resolution</h3><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">We've come to a conclusion for the incident, at least in the sense of stabilizing the environment. This phase is to close out any final actions or communications for the incident. It is important not to close the call right away, as there are things that can be important to keep a note of or required for future actions prior to problem management.</p><ul class="contains-task-list" style="box-sizing: border-box; margin-bottom: 0px !important; margin-top: 0px; padding-left: 2em; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li class="task-list-item" style="box-sizing: border-box; list-style-type: none;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">Final Actions - Incident Commander</strong><span> </span>needs to make any final actions now to complete stabilizing the environment. If any actions needed to be scheduled for a later time, discuss those now. (Scribe can take note in parallel).</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">Final Communication - Scribe</strong><span> </span>to collect final info regarding the incident (not RCA, that will come later). Final C.A.N, ELT and StatusPage updates to be sent out to respective stakeholders regarding the current state of the environment (expecting all is up and functional at this point).</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">Close incident/Bridge</strong><span> </span>- if not done already close the Pagerduty and/or Dynatrace alerts and close the Webex bridge. Also, close the Status Page incident and update the resolution (if not done in the previous step).</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span><strong style="box-sizing: border-box; font-weight: 600;">RCA &amp; Postmortem</strong><span> </span>- Scribe needs create an initial<span> </span><a href="https://github.com/ncr-bsp/rca/new/master/postmortems" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">RCA and Postmortem</a><span> </span>and close out with the help of the SMEs. Create a new PR for these new documents.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Once<span> </span><strong style="box-sizing: border-box; font-weight: 600;">RCA</strong><span> </span>is known, we adjust error budgets of applications to reflect accurate numbers.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Adjust any of the reported uptimes in StatusPage and PagerDuty if needed.</li><li class="task-list-item" style="box-sizing: border-box; list-style-type: none; margin-top: 3px;"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox" style="box-sizing: border-box; font: inherit; margin: 0px 0.2em 0.25em -1.6em; overflow: visible; padding: 0px; vertical-align: middle;"><span> </span>Close this Github issue.</li></ul>
</body>
</html>
